pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_Chen2020_plating/parameters.csv

```diff
@@ -29,6 +29,24 @@ Negative electrode specific heat capacity [J.kg-1.K-1],700,default,
 Negative electrode thermal conductivity [W.m-1.K-1],1.7,default,
```

Comment:
Are all these parameters presented in O'Kane 2022? If so, should we rename this set to O'Kane 2022 rather than Chen 2020 plating? We can still keep the ref to Chen 2020 in `pybamm.citations()` but I think the new name would make it clearer to the reader which is the main reference.

Reply:
I've gone further and deleted the entire Chen2020_plating chemistry. It was only there in the first place because none of the existing chemistries worked with the plating submodel. Now that OKane2022 exists, it's not needed anymore. OKane2020_Li_plating is still there as that is a published set of plating parameters, but none of the pre-defined chemistries use it.